modules/tinymce/src/plugins/textpattern/test/ts/browser/TextPatternPluginForcedRootBlockFalseTest.ts

```diff
@@ -0,0 +1,35 @@
+import {GeneralSteps, Pipeline, Step } from '@ephox/agar';
```

Comment:
Missing space here after `{`.

Switching to eslint would help us pick up on these automatically, but that's another days task 😛 

- [ ] Fix missing space